settings.php

```diff
@@ -67,6 +83,35 @@
        get_string('send_response_choices', 'logstore_xapi'),
        get_string('send_response_choices_desc', 'logstore_xapi'), 0));
 
+    // Control sending notifications.
+    $settings->add(new admin_setting_configcheckbox('logstore_xapi/enablesendingnotifications',
+        get_string('enablesendingnotifications', 'logstore_xapi'),
+        get_string('enablesendingnotifications_desc', 'logstore_xapi'), 1));
+
+    // Set threshold for sending notifications.
+    $settings->add(new admin_setting_configtext('logstore_xapi/errornotificationtrigger',
+        get_string('errornotificationtrigger', 'logstore_xapi'),
+        get_string('errornotificationtrigger_desc', 'logstore_xapi'), 10, PARAM_INT));
+
+    // Cohorts.
+    $settings->add(new admin_setting_heading('cohorts',
+        get_string('cohorts', 'logstore_xapi'),
+        get_string('cohorts_help', 'logstore_xapi')));
+
+    $cohorts = logstore_xapi_get_cohorts();
+    $arrcohorts = array();
+    foreach ($cohorts as $cohort) {
+        $arrcohorts[$cohort->id] = $cohort->name;
+    }
```

Comment:
Figured it out! It's because I didn't have any cohorts on the Moodle instance. Should probably account for this here.